### PR TITLE
[People] Data-Driven Heroes & Profiles

### LINKED ISSUE #7: [Profile] Build /profile/:slug

### LINKED ISSUE #8: [Heroes] Build /heroes page

### PR DESCRIPTION
**Pull request by charles-allen**
## 💯 DoD

- [x] All work is complete
- [x] Migrations: NA
- [x] Documentation: NA
- [x] Release notes: NA
- [x] Issues linked: resolves #7; resolves #8

## 📝 Summary

**General** (not sure why I didn't branch it)
- Ported some _settings_, _lint_, _build_ fixes (from CC)

**Heroes/Profile**
- Type updates:
  - `EventLog` (record of attendance & contribution)
  - `User` (profile, links, contribution)
  - `UserPrivate` (support multiple emails & networks)
  - `NewBase<T>`, `UpdateBase<T>` (support admin FieldValue & FieldValue for optionals)
- `dao.createUser`
- `people` test-data for seeder
- Update Heroes & Profile pages
  - Make both data-driven
  - Pretty colored SVG icons for user network links
  - Pretty shine effect on buttons

**Hint: You have to click to see the diffs for Heroes.html & Profile.html**

## 💡 More ideas

- Redesign profile page (single col on tablet & cleanup responsive grid)
- Multiple roles for one event

## 📸 Screenshots

Heroes

Profile

### CODE REVIEW

File: hosting/src/main.css [hunk 1]
```diff
@@ -1,16 +1,43 @@
...
 @tailwind components;
+@tailwind utilities;
 
-.tooltip {
-  @apply invisible absolute;
-}
+@layer utilities {
+  .tooltip {
+    @apply invisible absolute;
+  }
```
Comment (antonyharfield) [inline hosting/src/main.css:8]: All these @s are new to me too. I'm showing my css-age.

File: hosting/src/people/Profile/Profile.ts [hunk 2]
```diff
@@ -1,12 +1,69 @@
...
+    switch (network) {
+      case 'github': return `https://github.com/${id}`
+      case 'gitlab':return `https://gitlab.com/${id}`
+      case 'stackoverflow': return `https://stackoverflow.com/users/${id}`
+      case 'linkedin':return `https://www.linkedin.com/in/${id}/`
+      case 'twitter':return `https://twitter.com/${id}`
+      case 'facebook':return `https://www.facebook.com/${id}`
+      case 'line':return `https://line.me/R/ti/p/@${id}`
+      case 'email':return `mailto:${id}`
+    }
```
Comment (antonyharfield) [inline hosting/src/people/Profile/Profile.ts:53]: What's this? 😱

**Review by antonyharfield — APPROVED** [review]

File: hosting/src/main.css (hunk 1 above)
Comment (charles-allen) [reply hosting/src/main.css:8]: I think this is tailwind magic

File: hosting/src/people/Profile/Profile.ts (hunk 2 above)
Comment (charles-allen) [reply hosting/src/people/Profile/Profile.ts:53]: I might refactor it to a `Map` :p

**Pull request merged**